Sources/Metrics/Metrics.swift

```diff
@@ -71,6 +71,8 @@ extension Timer {
             self.recordSeconds(value)
         case .never:
             self.record(0)
+        @unknown default:
+            self.record(0)
```

Comment:
In most CI runs this cases the following error:

```
/code/Sources/Metrics/Metrics.swift:74:9: error: default will never be executed
        @unknown default:
        ^
[8/13] Compiling Metrics Metrics.swift
/code/Sources/Metrics/Metrics.swift:74:9: error: default will never be executed
        @unknown default:
        ^
```

Reply:
@ktoso Can you give an example of a CI config that causes this? I was about to open a PR to fix this because the enum is not frozen

Reply:
We'll have to run tests again to know since it's been so long that since the logs have been removed.

Reply:
info here: https://github.com/apple/swift-metrics/pull/137#issuecomment-1776719871 @AndrewHoos